internal/curator/durable/state/fb/state.fbs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2017 Western Digital Corporation or its affiliates. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+namespace internal.curator.durable.state.fb;
+
+// TractIDF is a 10-byte value that can refer to a tract or an RS chunk, just like common.TractID.
+// Declare it with shorts so flatbuffers doesn't pad for excessive alignment.
+struct TractIDF {
+  b1: ushort;
```

Comment:
Correct. I wish the fb schema used bits in their type names instead of this short/int/long stuff, but that's how it is.

Just FYI, the alternatives to this that don't work as well are:
1) Making a struct that holds a ulong plus a ushort: because the alignment of a ulong is 8 bytes, it allocates 16 bytes instead of 10.
2) Using [ubyte] instead of a struct: that takes an extra indirection because of the vector and also has to store the length somewhere.

Reply:
Thanks for updating the comment and the note on the alternatives.

Reply:
(Actually they do have type aliases with numbers of bits! I'll switch to those.)